source/items/platearmor.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Plate Armor
+layout: default
+---
+# Plate Armor
+
```

Comment:
Still need the infobox. Get the pics from [here](https://github.com/ldtteam/minecolonies/tree/version/main/src/main/resources/assets/minecolonies/textures/items) and upload 'em to the wiki